### DISCUSSION TITLE
Talos upgrade?

### DISCUSSION THREAD

**Discussion by rancherr** [hcloud-talos/terraform-hcloud-talos#55] [General]
Hei,
is there any Talos module which allow upgrading both Talos and k8s at MANY nodes?

**Comment by mrclrchtr** [comment]
Hi, not that I know of. Upgrades for talos are not so easy to implement with Terraform. My experience tells me that it is best to carry out the upgrades manually. They are quite easy with the talos cli.

**Reply by rancherr** [reply]
True, manually I know how to do it... but was thinking if there is any way to automate it - quite sure there are many ppl out there thinking about the same
